packages/agoric-cli/CHANGELOG.md

```diff
@@ -3,6 +3,59 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.17.0](https://github.com/Agoric/agoric-sdk/compare/agoric@0.16.0...agoric@0.17.0) (2022-09-20)
+
+
+### ⚠ BREAKING CHANGES
+
+* **legacy-smart-wallet:** remove dead package
```

Comment:
This package was never published so not really a breaking change. Doesn't seem to hurt though